docs/operating-scylla/procedures/cluster-management/replace-dead-node.rst

```diff
@@ -128,11 +118,11 @@ Procedure
          RACK:B1
          INTERNAL_IP:192.168.1.204
     
-         /192.168.1.203
+       /192.168.1.203
          generation:1553759866
          heartbeat:2147483647
-        HOST_ID:675ed9f4-6564-6dbd-can8-43fddce952gy
-         STATUS:shutdown,true
```

Comment:
If a node shutdown itself. It will be in shutdown status.

Reply:
okay.  I'll rerun the dtest and update the example output